crates/flox/src/commands/environment.rs

```diff
@@ -67,13 +129,12 @@ impl Edit {
                     .or(std::env::var("VISUAL"))
                     .context("no editor found; neither EDITOR nor VISUAL are set")?;
                 // TODO: check for interactivity before allowing the editor to be opened
-                let manifest_path = environment.manifest_path();
                 // Make a copy of the manifest for the user to edit so failed edits aren't left in
                 // the original manifest. You can't put creation/cleanup inside the `edited_manifest_contents`
                 // method because the temporary manifest needs to stick around in case the user wants
                 // or needs to make successive edits without starting over each time.
                 let tmp_manifest = NamedTempFile::new_in(&flox.temp_dir)?;
-                std::fs::copy(&manifest_path, &tmp_manifest)?;
+                std::fs::write(&tmp_manifest, environment.manifest_content()?)?;
```

Comment:
Why this change? Not that it really matters, but won't a copy potentially be faster?

Reply:
We don't have a `manifest_path` for remote and managed envs, and i wouldn't want to implement manifest_path with side effects.

If you'd benchmark it fs::copy will be faster since its likely doing the copy in kernel/fs level. Practically, its still just writing a couple kb.

Reply:
(this is non-blocking)
Aren't we copying the manifest locally even for remote and managed environments? So don't we have a `manifest_path`?

Reply:
only within the transaction as far as i understand, which we don't expose

Reply:
For both we should be storing the manifest in FLOX_META even when not running a transaction. I guess that's not something we have to commit to but I don't see that changing in the near future

Reply:
I was forgetting those will be bare repos